src/librustc/middle/cstore.rs

```diff
@@ -221,6 +221,9 @@ pub trait CrateStore {
     fn extern_mod_stmt_cnum_untracked(&self, emod_id: ast::NodeId) -> Option<CrateNum>;
     fn item_generics_cloned_untracked(&self, def: DefId, sess: &Session) -> ty::Generics;
     fn postorder_cnums_untracked(&self) -> Vec<CrateNum>;
+    fn maybe_process_path_extern_untracked(
```

Comment:
Can you rename this and everything else to `maybe_load_extern_crate`(`_untracked`)?

Reply:
Fixed, thanks!